shell/platform/android/io/flutter/plugin/editing/FlutterTextUtils.java

```diff
@@ -186,4 +186,124 @@ public int getOffsetBefore(CharSequence text, int offset) {
 
     return offset - deleteCharCount;
   }
+
+  /**
+   * Gets the offset of the next character following the given offset, with consideration for
+   * multi-byte characters.
+   */
```

Comment:
Can you link the android logic that is also linked in `getOffsetBefore()`? Will be helpful for future readers.